scripts/services/migrations.ts

```diff
@@ -4,7 +4,8 @@ export const migration: T.ExpectedExports.migration = compat.migrations
   .fromMapping( {
       // 24.0.4 No migration needed 
       // 24.0.4.1 No migration needed 
-      // 24.0.4.2 No migration needed 
+      // 24.0.4.2 No migration needed
+      // 24.0.4.3 No migration needed  
     },
-    "24.0.4.3" 
```

Comment:
update this version as well to match what the one in manifest is changed to, per above comment

Reply:
done